### PR TITLE
[DOC] Add tutorial for infinite scrolling

### PR DESCRIPTION
**Pull request by georgringer**

### CODE REVIEW

File: Documentation/Tutorials/BestPractice/InfiniteScroll/Index.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+=====================
+
+Using ajax for the pagination can be achieved by using little bit of JavaScript
+
+.. hint::
+   There are various discussions if endless scrolling and ajax based pagination
+   is actually a good idea as there are also some disadvantages as well.
+
+This tutorial uses the implementation of https://infinite-scroll.com/.
+**Be aware** that this code requires buying a license!
```
Comment (DavidBruchmann) [inline Documentation/Tutorials/BestPractice/InfiniteScroll/Index.rst:16]: **Be aware** that this code requires buying a `license <https://infinite-scroll.com/#license>`__ !

Comment (DavidBruchmann) [reply Documentation/Tutorials/BestPractice/InfiniteScroll/Index.rst:16]: Open source projects might use infinite-scroll for free:

> Open source license
> If you are creating an open source application under a license compatible with the [GNU GPL license v3](https://www.gnu.org/licenses/gpl-3.0.html), you may use Infinite Scroll under the terms of the GPLv3. [Read more about Infinite Scroll open source licensing](https://infinite-scroll.com/license.html).

Comment (georgringer) [reply Documentation/Tutorials/BestPractice/InfiniteScroll/Index.rst:16]: Yes but most Webseite für clients are not open source.

Comment (DavidBruchmann) [reply Documentation/Tutorials/BestPractice/InfiniteScroll/Index.rst:16]: The sentence is not accurate though and two things could easily be clarified with my proposition

- which license had to be bought.
- buying a license is not always required.

For this my second comment with the quote from the website never even had to be added.

Comment (georgringer) [reply Documentation/Tutorials/BestPractice/InfiniteScroll/Index.rst:16]: thanks. adopted

**Pull request merged**